docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -56,6 +56,28 @@ cluster.initial_master_nodes:
   - master-node-hostname
 --------------------------------------------------
 
+[NOTE]
+==================================================
+The node names used in this list must exactly match the `node.name` properties
+of the nodes. If your node names are a fully-qualified domain name such as
```

Comment:
Wdyt about:

By default the `node.name` setting is resolved to the machine hostname. If the host name is fully-qualified domain name such as ...

Reply:
Sure; we mention the default a few lines above but it seems fine to mention it again. I did this in eacb7640de0.